GripOpGras2.Specs/Features/RationAlgorithmV1.feature

```diff
@@ -0,0 +1,102 @@
+
+The first approach devised to create a ration for a herd
+
+Scenario: Create a ration for a herd, when having two roughage products
+	Given I have the roughage product <roughage1> that contains <protein-roughage1> g protein, and <vem-roughage1> VEM
+	And I have the roughage product <roughage2> that contains <protein-roughage2> g protein, and <vem-roughage2> VEM
+	And I have a herd with <herd-size> cows in it, which have taken in <grass-intake> kg dm grass
+	And each kg dm grass contains <vem-grass> VEM and <protein-grass> g protein
+	And my herd has produced <lmilk-produced> liters of milk
+	When I let Grip op Gras 2 create a ration
+    Then the ration should contain between <dm-roughage-ration-min> and <dm-roughage-ration-max> kg dm of roughage products
+    And the ration should contain between <dm-supplementary-ration-min> and <dm-supplementary-ration-max> kg dm of supplementary products
+    And the ration should contain between <protein-ration-min> and <protein-ration-max> g protein
+    And the ration should contain between <vem-ration-min> and <vem-ration-max> VEM
+	And the ration must contain <grass-intake> kg of grass
+Examples: 
+| roughage1 | protein-roughage1 | vem-roughage1 | roughage2 | protein-roughage2 | vem-roughage2 | herd-size | grass-intake | vem-grass | protein-grass | lmilk-produced | dm-roughage-ration-min | dm-roughage-ration-max | dm-supplementary-ration-min | dm-supplementary-ration-max | protein-ration-min | protein-ration-max | vem-ration-min | vem-ration-max |
+
+
+Scenario: Create a ration for a herd, when having three roughage products
+	Given I have the roughage product <roughage1> that contains <protein-roughage1> g protein, and <vem-roughage1> VEM
+	And I have the roughage product <roughage2> that contains <protein-roughage2> g protein, and <vem-roughage2> VEM
+	And I have the roughage product <roughage3> that contains <protein-roughage3> g protein, and <vem-roughage3> VEM
+	And I have a herd with <herd-size> cows in it, which have taken in <grass-intake> kg dm grass
+	And each kg dm grass contains <vem-grass> VEM and <protein-grass> g protein
+	And my herd has produced <lmilk-produced> liters of milk
+	When I let Grip op Gras 2 create a ration
+    Then the ration should contain between <dm-roughage-ration-min> and <dm-roughage-ration-max> kg dm of roughage products
+    And the ration should contain between <protein-ration-min> and <protein-ration-max> g protein
+    And the ration should contain between <vem-ration-min> and <vem-ration-max> VEM
+	And the ration must contain <grass-intake> kg of grass
+Examples: 
+| roughage1 | protein-roughage1 | vem-roughage1 | roughage2 | protein-roughage2 | vem-roughage2 | roughage3 | protein-roughage3 | vem-roughage3 | herd-size | grass-intake | vem-grass | protein-grass | lmilk-produced | dm-roughage-ration-min | dm-roughage-ration-max | protein-ration-min | protein-ration-max | vem-ration-min | vem-ration-max | kg-roughage3-min | kg-roughage3-max |
+
```

Comment:
shouldn't the scenario's without examples be commented out, or shouldn't there be a documentation why it is not used?